### PR TITLE
Reminder form edits

### PR DESCRIPTION
**Pull request by Mhorvilleur13**
# Context

This PR was made to resolve comments from the previous PR

# Tests

Describe what tests have you done
No Changes have been made to the UI. 

Attach screenshots, before and after

### CODE REVIEW

File: src/types/recurrence-config.ts [hunk 1]
```diff
@@ -11,6 +11,6 @@ export enum RecurrenceType {
   DAILY = "DAILY",
   WEEKLY = "WEEKLY",
   MONTHLY = "MONTHLY",
-  YEARLY = "YEARLY",
+  ANUALLY = "ANUALLY",
```
Comment (guzzur) [inline src/types/recurrence-config.ts:14]: Oh, yeah, my bad, thank you :facepalm:

File: src/components/reminder-form/reminder-form.tsx [hunk 2]
```diff
@@ -1,18 +1,83 @@
...
+      <div className="card mt-3 text-secondary" >
+        <h6>Is this a recurring task?</h6>
+        <label htmlFor="yes">Yes</label>
+        <input id="yes" value="yes" type="radio" onChange={e => setRecurring(e.target.checked === true ? true : false)} />
+        <label htmlFor="no">No</label>
+        <input id="no" value="no" type="radio" onChange={e => setRecurring(e.target.checked === true ? false : true)} />
+      </div>
+      <div className="card mt-3 text-secondary">
+        <h6>How often would would like to be reminded?</h6>
+        {/* <select onChange={e => setReccuringFrequency(RecurrenceType[e.target.value as keyof typeof RecurrenceType])} id="reminder-frequency" name="reminder"> */}
```
Comment (guzzur) [inline src/components/reminder-form/reminder-form.tsx:69]: No commented out code please

**Review by guzzur — COMMENTED** [review]
Great work. Thank you!

A few comments:
1. None of the commit messages match the convention "When the commit is applied it will X"
2. TBH I don't get why did you create this PR before merging previous one. Now you have the same code twice for me to review and for you to merge. Do you still need https://github.com/Mhorvilleur13/Reminder-App/pull/13 or does this PR obsoletes it? So many questions, so little answers...

File: public/index.html [hunk 3]
```diff
@@ -1,18 +1,27 @@
...
+<body>
+  <noscript>You need to enable JavaScript to run this app.</noscript>
+  <div id="root"></div>
+</body>
+<script src="https://code.jquery.com/jquery-3.2.1.slim.min.js"
+  integrity="sha384-KJ3o2DKtIkvYIK3UENzmM7KCkRr/rE9/Qpg6aAZGJwFDMVNA/GpGFF93hXpG5KkN" crossorigin="anonymous"></script>
+<script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.12.9/umd/popper.min.js"
+  integrity="sha384-ApNbgh9B+Y1QKtv3Rn7W3mgPxhU9K/ScQsAP7hUibX39j7fakFPskvXusvfa0b4Q" crossorigin="anonymous"></script>
+<script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/js/bootstrap.min.js"
+  integrity="sha384-JZR6Spejh4U02d8jOt6vLEHfe/JQGiRRSQQxSfFWpi1MquVdAyjUar5+76PVCmYl" crossorigin="anonymous"></script>
```
[the remark this answers is not in the document]
Comment (Mhorvilleur13) [reply public/index.html:25]: I want to include a collapse if the task is recurring.

File: src/components/reminder-form/reminder-form.tsx (hunk 2 above)
Comment (Mhorvilleur13) [reply src/components/reminder-form/reminder-form.tsx:69]: I know sorry. I missed that.

File: src/components/reminder-form/reminder-form.tsx [hunk 4]
```diff
@@ -3,33 +3,56 @@ import React, { FormEvent, useState } from "react";
 
 const Form = ({ }: {}) => {
   const [taskName, setTaskName] = useState('');
-  const [recurring, setRecurring] = useState(false);
+  const [taskMessage, setTaskMessage] = useState('');
+  const [reminderType, setReminderType] = useState('');
+  const [reminderDate, setReminderDate] = useState('');
+  const [recurring, setRecurring] = useState(true);
+  const [reccuringFrequency, setReccuringFrequency] = useState('');
+  const [reminderBeforeMs, setMs] = useState('');
```
[the remark this answers is not in the document]
Comment (Mhorvilleur13) [reply src/components/reminder-form/reminder-form.tsx:15]: I kept getting an error when I tried to set it to a number.

File: public/index.html (hunk 3 above)
[the remark this answers is not in the document]
Comment (guzzur) [reply public/index.html:25]: I didn't see a commit adding or using the collapse component. This code was added in totally unrelated commit.

**Pull request merged**